docs/index.md

```diff
@@ -130,31 +132,30 @@ The URL will be (update the GitHub username variable):
    ```  
 
 Open this in your browser, you should see:
-![Default Page Template](/images/DefaultPageTemplate.png)
+![Default Page Template](images/DefaultPageTemplate.png)
 
 ### Customise application
 
-We are going to update the application by changing the home page.
-
-1. Open the 'home.njk' file inside the 'src/main/views'
-2. Edit the html heading to change the webpage's text
+We are going to update the application by changing the text on the home page.
 
-Before:
+1. Go to the Github repo you created at the beginning of this tutorial.
+2. Open the file 'home.njk' under 'src/main/views'.
+3. Edit the html heading to change the webpage's text, such as this [example](https://github.com/hmcts/labs-zcwalkthrough-nodejs/pull/3/files).
+4. Ask someone in your team to review your pull request and then merge it.
+5. Run the Jenkins pipeline against the master branch (this will trigger automatically on the production Jenkins instance).
+6. Reload your application in your browser and check it now shows your change to the heading:
+![Hello World](images/HelloWorld.png)
 
-  ```yaml
-  <h1 class="govuk-heading-xl">Default Page Template</h1>
-   ```
+## Feedback
 
-After:
+[comment]: <> (As of March 2022)
+This is a new way of onboarding developers that we are trying out.
+If you could provide feedback it would really help us improve it for others.
+The [survey](https://forms.office.com/r/P2YbcLVAr4) has 4 questions and will only take you a couple of minutes to complete.
 
-   ```yaml
-  <h1 class="govuk-heading-xl">Hello World!</h1>
-   ```
+If you've found a problem with the guide please [report an issue](https://github.com/hmcts/golden-path-java/issues) instead, or you can create a pull request to correct it yourself.
```

Comment:
The issues URL would need changed to reflect the golden-path-nodejs repo: https://github.com/hmcts/golden-path-nodejs/issues

Reply:
```suggestion
If you've found a problem with the guide please [report an issue](https://github.com/hmcts/golden-path-nodejs/issues) instead, or you can create a pull request to correct it yourself.
```